docs/DOCKER.md

```diff
@@ -110,17 +110,16 @@ to expose RPC ports, use `btcd` or `bitcoind`, or add additional chains.
 
 ## LND Development and Testing
 
-To test the Docker production image locally, run the following from
-the project root:
+To test the Docker production image locally, run the following from the project root:
 
 ```shell
 ⛰  docker build . -t myrepository/lnd:master
 ```
 
-To choose a specific branch or tag instead, use the "checkout" build-arg.  For example, to build the latest commits in master:
+To choose a specific [branch](https://github.com/lightningnetwork/lnd/tags) or [tag](https://hub.docker.com/r/lightninglabs/lnd/tags?page=1&ordering=last_updated) instead, use the "checkout" build-arg.  For example, to build the latest commits in master:
```

Comment:
Branches are separate from tags, so that link should point to https://github.com/lightningnetwork/lnd/branches instead?